UI/src/app/services/configuration.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from '@angular/core';
+import { IStorage } from '../models/interface/storage.interface';
+import { Configuration } from '../models/configuration';
+import { BridgeService } from './bridge.service';
+import { BrowserSessionConfigurationStorage } from '../models/implementation/configuration-session-storage';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { StoreService } from './store.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConfigurationService {
+
+    configStorageService: IStorage<Configuration>;
+    configurations = [];
+    private snakbarOptions = {
+        duration: 3000
+      };
+
+    constructor(
+        private bridgeService: BridgeService,
+        private snakbar: MatSnackBar,
+        private storeService: StoreService
+    ) {
+        this.configStorageService = new BrowserSessionConfigurationStorage('configurations');
+        this.configurations = [...Object.values(this.configStorageService.configuration)];
+    }
+
+    openConfiguration(configurationName: string) {
+        const config = this.configStorageService.open(configurationName);
+        if (config) {
```

Comment:
probably some logic is omited there. In other case we need to remove empty if

Reply:
corrected